src/planwise/client/handlers.cljs

```diff
@@ -13,9 +13,24 @@
    (playground/fetch-facilities-with-isochrones :immediate (:playground db/initial-db))
    db/initial-db))
 
+(defmulti navigate-handler (fn [db [_ page & _]] page))
+
+(defmethod navigate-handler :projects [db [_ page & params]]
+  (let [id (js/parseInt (first params))
+        db (assoc db
+             :current-page page
+             :page-params params)]
+    (if (not= id (get-in db [:projects :current :id]))
+      (do
+        (dispatch [:projects/load-project id])
```

Comment:
Loading the project is _requested_ by firing an event, which is handled appropriately. Should this be handled by actually setting the appropriate db fields and calling the API here, rather than delegating that to the handler?